aldjemy/core.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, pool, alias):
 
         self.alias = alias
         self.wrap = False
-        #pool.dispatch.first_connect.exec_once(self.connection, self)
+        pool.dispatch.first_connect.exec_once(self.connection, self)
```

Comment:
I don't know why it was commented out, and i'm not sure if uncommenting it breaks anything either. But it works in my project so far with that line uncommented.